### DISCUSSION TITLE
Support for Swoole asynchronous style

### DISCUSSION THREAD

**Discussion by wisonlau** [apache/skywalking#12812] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

PHP (apache/skywalking-php)

### What happened

https://github.com/apache/skywalking/issues/12806
This fix only supports coroutine style and lacks asynchronous style. Currently, I am using asynchronous style
https://wiki.swoole.com/zh-cn/#/http_server
<img width="985" alt="image" src="https://github.com/user-attachments/assets/80155c1b-0728-4d15-aeb2-7baa8a20c9e6">

### What you expected to happen

Hope to support asynchronous style

### How to reproduce

hyperf frame catalogue: config/server.php
```
'mode' => SWOOLE_BASE,
'servers' => [
        [
            'name' => 'http',
            'type' => ServerInterface::SERVER_HTTP,
            'host' => env('HTTP_HOST', '0.0.0.0'),
            'port' => (int) env('HTTP_PORE', 9501),
            'sock_type' => SWOOLE_SOCK_TCP,
            'callbacks' => [
                Event::ON_REQUEST => [Hyperf\HttpServer\Server::class, 'onRequest'],
            ],
        ],
]
```

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wisonlau** [comment]
@jmjoy have a look

**Reply by jmjoy** [reply]
O

**Comment by wu-sheng** [comment, marked as answer]
Added through https://github.com/apache/skywalking-php/pull/121
